<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Przekszta≈Çcenia 2D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Ekran" name="ekran" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="x_slider">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-500</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="y_slider">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-500</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sx_slider">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sy_slider">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="shx_slider">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="shy_slider">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="obrot_slider">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="inter_check">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Interpolacja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translacja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Skalowanie</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotacja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pochylenie</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ekran</class>
   <extends>QWidget</extends>
   <header location="global">ekran.h</header>
   <container>1</container>
   <slots>
    <signal>setCurrentAlpha(int)</signal>
    <slot>alphaSliderChange(int)</slot>
    <slot>warstwaButton()</slot>
    <slot>alphaSliderChange()</slot>
    <slot>setAlpha(int)</slot>
    <slot>blendMode()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
